EDSAnalytics/EDSTest/README.md

```diff
@@ -0,0 +1,105 @@
+# EDS Analytics Sample and Test
+
+[![Build Status](https://PLACE_BUILD_STATUS_LINK_HERE)]
```

Comment:
We can add this if you'd like, but first I think the `azure-pipeline.yml` file needs to be moved up one level in the folder hierarchy.